compose.yaml

```diff
@@ -3,6 +3,9 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.dev
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
```

Comment:
Is `/usr/src/app/node_modules` this needed? Because it will be mounted as part of the project on `.`?

Reply:
Hey @eddiejaoude,

The above config is correct. Both are different. This one, `.:/usr/src/app`, mounts the entire current directory. Second, we’re not mounting again; instead, we’re preventing `node_modules` from being overridden in the container and the container uses its dependencies using `/usr/src/app/node_modules`.

Reply:
oh I see, thank you for explaining 👍